topics/journalpost.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: journalpost
+  namespace: bidrag
+  labels:
+    team: bidrag
+spec:
+  pool: {{kafka-pool}}
+  config:
+    cleanupPolicy: compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3
+    retentionBytes: -1
+    retentionHours: 72
+  acl:
+    - team: bidrag
```

Comment:
Bør vel også legge inn readwrite for bidrag-dokument-arkiv og read forbidrag-arbeidsflyt?

Reply:
jeg synes ikke vi skal legge inn noe konfigurerering før vi faktisk begynner å implementere på det